linux_docker_resources/Dockerfile-CentOS

```diff
@@ -92,13 +88,15 @@ RUN yum install \
   python36-catkin_pkg \
   python36-empy \
   python36-lark-parser \
+  python36-lxml \
   python36-mock \
   python36-numpy \
   python36-psutil \
   python36-pyflakes \
   python36-pygraphviz \
   python36-pytest \
   python36-PyYAML \
+  python36-qt5-devel \
```

Comment:
Was this being pulled in by something else before or is this a new dependency on this platform?

Reply:
It [showed up](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2019-f5aa6f723a) in EPEL 7 stable about a month after I made this Dockerfile, and I just now noticed.

Once the [packaging bug](https://bugzilla.redhat.com/1742219) upstream is resolved, we'll be able to use it to build all of the `rqt_*` packages (I'll follow up with another PR). That will leave only a single demo package that can't be built on CentOS 7 due to the OpenCV version.

This will make the `rqt_reconfigure` tests pass though.